modules/nodes-pods-autoscaling-policies.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-policies_{context}"]
+= Scaling policies
+
+The `autoscaling/v2beta2` API allows you to create _scaling policies_ that control how the {product-title} horizontal pod autoscaler (HPA) scales Pods. Scaling policies allow you to restrict the rate that HPAs scale Pods up or down by setting a specific number or specific percentage to scale in a specified period of time. You can also define a _stabilization window_ which uses previously computed desired states to control scaling if the metrics are fluctuating. You can create multiple policies for the same scaling direction and determine which policy is used first, based on the amount of change. You can also restrict the scaling by timed iterations. The HPA scales pods during an iteration, then performs scaling, as needed, in further iterations. 
+
+By default, the scaling policies are implemented when the number of Pods associated with an HPA is greater than 40.
+
+.Sample HPA object with a scaling policy
+[source, yaml]
+----
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-resource-metrics-memory
+  namespace: default
+spec:
+  behavior:
+    scaleDown:
+      policies: <1>
+      - type: Pods <2>
+        value: 4 <3>
+        periodSeconds: 60 <4>
+      - type: Percent
+        value: 10
+        periodSeconds: 60
+      selectPolicy: Min <5>
+      stabilizationWindowSeconds: 300 <6>
+    scaleUp:
+      policies:
+      - type: Pods
+        value: 5
+        periodSeconds: 70
+      - type: Percent
+        value: 12
+        periodSeconds: 80
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+...
+----
+
+<1> Scaling policies allows you to control autoscaling by the number of pods or a percentage over a period of seconds for scaling up or down. 
+<2> The `type` parameter determines if the policy scales by a specific number of Pods or a percentage of Pods during each iteration. The default value is `Pods`.
+<3> The `value` parameter determines the amount of scaling, either the number of Pods or percentage of Pods during each iteration. The default value is `4`.
```

Comment:
There are 4 cases for `value`. Scale up - pods, Scale up - percent, Scale down - pods, and Scale down - percents. There is no default for Scale down - pods (which is the `value` being annotated here). The defaults for Scale down - percent and Scale up - percent are both 100%.  The defaults for Scale up - pods is 4.

Reply:
@mburke5678 it seems the comments from Joel did not get addressed: 
 The defaults for Scale up - pods is 4.
There is no default for Scale down